frontend/src/helpers/validation.js

```diff
@@ -0,0 +1,11 @@
+import { helpers } from 'vuelidate/lib/validators';
+
+export const slackNickValidator = helpers.regex(
+  'slackNickValidator',
+  /^[0-9A-Za-zżźćńółęąśŻŹĆĄŚĘŁÓŃ,.';\-_/()[\]{}]*$/
```

Comment:
I hate to be THAT guy but I would use: `ąćęłńóśźżĄĆĘŁŃÓŚŹŻ` - just for the sake of my OCD (and readability).

Reply:
Done